spec/lib/search_spec.rb

```diff
@@ -2176,6 +2176,18 @@ def search_with_bookmarks
       expect(Search.execute("Topic max_views:150").posts.map(&:id)).to eq([post.id])
     end
 
+    it "can order by likes" do
+      raw = "Foo bar lorem ipsum"
+      topic = Fabricate(:topic)
+      post1 = Fabricate(:post, topic:, raw:, like_count: 1)
+      post2 = Fabricate(:post, topic:, raw:, like_count: 2)
+      post3 = Fabricate(:post, topic:, raw:, like_count: 3)
```

Comment:
dunno why but this shorthand syntax takes more time getting used to for me 😅

Reply:
I'm loving it personally 😍

Reply:
It's confusing for people who write a lot of js, because we would expect: `Fabricate(:post, topic, raw, like_count: 3)`